include/rocksdb/db.h

```diff
@@ -374,6 +374,20 @@ class DB {
                     keys, values);
   }
 
+  virtual void MultiGet(const ReadOptions& options,
+                        const std::vector<ColumnFamilyHandle*>& column_family,
+                        const std::vector<Slice>& keys, PinnableSlice* values,
+                        Status* statuses) {
```

Comment:
That's a good idea. I didn't think about it. If we make them also pointers to array, it will allow callers to use the stack instead of vector allocations